modules/tinymce/src/plugins/link/main/ts/api/Commands.ts

```diff
@@ -1,17 +1,16 @@
 import Editor from 'tinymce/core/api/Editor';
 
-import * as Actions from '../core/Actions';
+import * as Dialog from '../ui/Dialog';
 import * as Options from './Options';
 
 const register = (editor: Editor): void => {
-  editor.addCommand('mceLink', () => {
-    if (Options.useQuickLink(editor)) {
-      // Taken from ContextEditorEvents in silver. Find a better way.
+  editor.addCommand('mceLink', (_ui, dialog?: boolean ) => {
```

Comment:
There was a reason I mentioned using an object on slack for this, it's because this isn't extensible. This locks us in heavily and if in future we need to pass 2 different things in we can't without doing a breaking change. So generally we try to use objects for these use cases as they can be extended in a non-breaking way.

- [ ] Change the command value to be an object with a dialog or similar property instead